js/main.js

```diff
@@ -1,3 +1,24 @@
+var tabs = document.querySelectorAll('.tab');
+var allPanels = document.querySelectorAll(".schedule");
```

Comment:
here is the same about  `const` and you use double quotes just here, in the other strings you have been using the single ones, is better to keep using one style everywhere.